### ISSUE TITLE
Add User page 

### ISSUE THREAD

**Issue by wood-crafter** [wood-crafter/my-tinder#42]

**Comment by wood-crafter** [comment]
@hoangph271 
Please make me an API to get user info -> Could be like this:
    email: 'phanhung2702@gmail.com',
    phone: '0366513328',
    address: 'HN',
    facebook: 'dummyFacebookLink',
    instagram: 'dummyInstagramLink'
    bio: 'dummyBio',
    type: 'dummyType',
    interest: 'dummyInterest',
    jobTitle: 'dev'

**Comment by hoangph271** [comment]
@wood-crafter dude, there's this API here:  
`https://sneu.date/api/v1/profiles`
The response body:

```json
{
  "statusCode": 200,
  "item": {
    "username": "hbp",
    "title": "hbp",
    "avatarUrl": "https://alpha-sneu.xyz/static/svg/sneu.svg",
    "description": "Welcome, dude...!"
  }
}
```

**Comment by wood-crafter** [comment]
@hoangph271 
Got this, told you before:
![image](https://user-images.githubusercontent.com/59172319/193728947-6e0e7749-47e8-4e0c-93e7-7bb1c8793ef8.png)
Also, please refer to extra infor in response as above

**Comment by hoangph271** [comment]
@wood-crafter yeah and I explained that you need to sign in and fetch the API with JavaScript...!

**Comment by wood-crafter** [comment]
> @wood-crafter yeah and I explained that you need to sign in and fetch the API with JavaScript...!

@hoangph271 
I should fetch with bearer token got from sign in response, ain't it?

**Comment by hoangph271** [comment]
@wood-crafter yeah, use the Authorization header:  
`Authorization: Bearer <token>`
